### PR TITLE
Add links to flyers to site

### PR DESCRIPTION
**Pull request by inglesp**

### CODE REVIEW

File: content/flyers.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+type: page
+title: Flyers
+slug: flyers
+---
+
+Here are some flyers to promote the conference.
+Please feel free to share widely.
+
+* [Open Day](https://raw.githubusercontent.com/PyconUK/2016.pyconuk.org/master/resources/flyers/PyCon UK 2016 Open Day.pdf)
```
Comment (meatballs) [inline content/flyers.md]: Shouldn't these be relative URLs? Very difficult to test like this.

Comment (inglesp) [inline content/flyers.md]: I couldn't work out how to get wok to do the right thing with PDFs last night, but it was obvious when I looked at it again just now, so this should be fixed.

**Comment by meatballs** [comment]
the flyers page itself looks fine now but where is the link to it from anywhere else? Is there supposed to be one?

**Comment by inglesp** [comment]
@meatballs, thanks for taking a look.  No, this isn't linked to from anywhere else yet -- @evildmp had just asked that these be made available on the website so that they can be easily distributed.

**Pull request merged**